### PR TITLE
Option to add original csv line as additional_data

### PR DESCRIPTION
**Pull request by deanpemberton**
This helped us during the initial deployment and debug phase.  Not intended for production deployment.

### CODE REVIEW

**Comment by wesyoung** [comment]
ty!

**Comment by mdavis332** [comment]
@wesyoung it looks like this commit added the `self.add_orig = True` by default to the CSV parser, which means the additional_data field will include the full CSV line. Dean mentioned he thought this should only be used for testing, but it got merged to master as default. 

Any reason not to remove that `self.add_orig` line from the CSV parser which defaults to `False`, and aligns it with the other parsers?

File: csirtg_smrt/parser/zcsv.py [hunk 1]
```diff
@@ -8,5 +8,6 @@ def __init__(self, *args, **kwargs):
         super(Csv, self).__init__(*args, **kwargs)
 
         self.pattern = re.compile(",")
+        self.add_orig= True
```
Comment (dwdixon) [inline csirtg_smrt/parser/zcsv.py:11]: @wesyoung FWIW, I'd be inclined (based on discovering this and our experiences with this being a surprise problem) to agree with the comment [here](https://github.com/csirtgadgets/csirtg-smrt-v1/pull/265#issuecomment-651466201) that it makes a lot of sense to remove this line and let it default to False for all parsers.  I think letting folks enable this only when needing to do some specific troubleshooting on a per-parser basis makes the most sense, this was generating lots of errors on our cluster and almost made it past us prior to moving into production.

Comment (mdavis332) [reply csirtg_smrt/parser/zcsv.py:11]: Went ahead and created PR #292 to address this.

**Pull request merged**